containers/FeedList/components/RecommendProfile.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styled from "@emotion/styled";
+import colors from "@/styles/colors";
+
+interface Props {
+    userImage: string;
+    userName: string;
+    isFollowed: boolean;
+    onFollow: () => void;
+}
+
+const RecommendProfile = ({userImage, userName, isFollowed, onFollow}: Props) => {
+    const profileImage = userImage ? (
+        <ProfileImage src={userImage}/>
+    ) : (
+        <DefaultProfileImage/>
+    );
+
+    return (<Row>
+        {profileImage}
+        <Test>
+            <NameText>
+                {userName}
+            </NameText>
+        </Test>
+        <FollowButton onClick={onFollow} isFollowed={isFollowed}>
+            팔로우
+        </FollowButton>
+    </Row>);
+}
+
+export default RecommendProfile
+
+const Row = styled.section`
+  display: flex;
+  align-items: center;
+  padding: 8px 0;
+  gap: 8px;
+`;
+
+// TODO : 텍스트 오버플로우 문제 해결
+const NameText = styled.p`
+  font-weight: 400;
+  color: #f6f6f6;
+`;
```

Comment:
```suggestion
const NameText = styled.p`
  font-weight: 400;
  color: #f6f6f6;
  text-overflow: ellipsis;
`;
```
text-overflow 추가하고 text자체에 width를 주거나 감싸고 있는 부분을 width로 크기 고정을 하면 해결될겁니다.